content/md/what-is/what-is-an-asset.md

```diff
@@ -0,0 +1,51 @@
+---
+id: what-is-an-asset
+themes: first-steps, akeneo-concepts, manage-your-assets
+title: What is an **asset**?
+popular: false
+ee-only: true
+related: assets-transformation, upload-assets, work-with-assets
+---
+
+# Definition of an asset
+
+You will use an asset whenever you want to enrich your products with files. These files can be, for example, pictures or user manuals of your products.
+
+::: ee
+This feature is only available in the Entreprise Edition.
+:::
+
+One asset is composed of several files depending on if you need a different version of this asset for each of your locales or channels.
+
+![Asset](../img/Settings_What-is-an-asset.svg)
+
+An asset has a unique code which is mandatory and optional properties :
+- A set of files: one or several reference files and their variations per channel,
+- A description,
+- An end date of use,
+- Some tags (to qualify the asset and find it quickly).
+
+As stated above, an asset can be localizable. In this case, you will have as many reference files than activated locales. It is very useful when you need, for example, one PDF file for your manual in English and another one in French.
+
+To finish, you can define different transformations of these assets according to your channel needs, i.e. a picture with different sizes depending on the channels. To learn more about these transformations, you can go [here](assets-transformation.html).
```

Comment:
To unleash the power of our Product Asset Manager (PAM), you will need to upload asset transformations for your channels (see our dedicated article: [About the asset transformations](assets-transformation.html) so the PIM generates variations based on the reference file.
Let's say you upload a HD image as a reference file, you can decide to reduce its resolution and/or size for your e-commerce channel, and do the same for the mobile channel. This is what asset transformation are used for. 

:::warning
The PIM can natively only generate variations for image files, so if you upload a PDF file or a Video as a reference file, no variations will be generated. But you can add your own new transformations! To do so, check our [technical cookbook](https://docs.akeneo.com/latest/manipulate_pim_data/product_asset/add_new_transformation.html)  
:::